### PR TITLE
[13.0][MIG] website_sale_order_type: Migration to 13.0

### PR DESCRIPTION
**Pull request by oihane**

### CODE REVIEW

File: website_sale_order_type/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+This module allows `sale_order_type <https://github.com/OCA/sale-workflow/tree/10.0/sale_order_type>`__ to work with website_sale.
```
Comment (chienandalu) [inline website_sale_order_type/readme/DESCRIPTION.rst]:
```suggestion
This module allows `sale_order_type <https://github.com/OCA/sale-workflow/tree/13.0/sale_order_type>`__ to work with website_sale.
```

File: website_sale_order_type/models/sale_order.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        self,
+        product_id=None,
+        line_id=None,
+        add_qty=0,
+        set_qty=0,
+        attributes=None,
+        **kwargs
+    ):
+        self.ensure_one()
+        res = super(SaleOrder, self)._cart_update(
```
Comment (chienandalu) [inline website_sale_order_type/models/sale_order.py]:
```suggestion
        res = super()._cart_update(
```

**Review by chienandalu — APPROVED** [review]
:+1: Minor tweaks:

**Comment by chienandalu** [comment]
@joao-p-marques please take a look

File: website_sale_order_type/tests/test_website_sale_order_type.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        self.partner.sale_type = self.sale_type
+        existing_orders = self.env["sale.order"].search([])
+        # In frontend, create an order
+        tour_prefix = "odoo.__DEBUG__.services['web_tour.tour']"
+        self.phantom_js(
+            "/shop",
+            tour_prefix + ".run('website_sale_order_type_tour')",
+            tour_prefix + ".tours.website_sale_order_type_tour.ready",
+            login="admin",
+        )
```
Comment (chienandalu) [inline website_sale_order_type/tests/test_website_sale_order_type.py]: There's now a simpler way to invoke the tours with this new wrapper:
```suggestion
        self.start_tour("/shop", "website_sale_order_type_tour", login="admin")
```

File: website_sale_order_type/tests/test_website_sale_order_type.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                "name": "Test Sales Order",
+                "code": "sale.order",
+                "prefix": "TSO",
+                "padding": 3,
+            }
+        )
+        self.journal = self.env["account.journal"].search(
+            [("type", "=", "sale")], limit=1
+        )
+        self.warehouse = self.env.ref("stock.stock_warehouse_shop0")
```
Comment (chienandalu) [inline website_sale_order_type/tests/test_website_sale_order_type.py]: As I see from the logs, this warehouse missmatches the one from the order company. Maybe that's avoiding the tour to finish correctly

**Review by chienandalu — CHANGES_REQUESTED** [review]
Review tests

File: website_sale_order_type/tests/test_website_sale_order_type.py (hunk 4 above)
Comment (oihane) [reply website_sale_order_type/tests/test_website_sale_order_type.py]: Thanks @chienandalu, I finally manage to get the test working.

File: website_sale_order_type/__manifest__.py [hunk 5]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Website sale order type",
     "summary": "This module allows sale_order_type to work with website_sale.",
-    "version": "12.0.1.1.0",
+    "version": "13.0.1.1.0",
```
Comment (joao-p-marques) [inline website_sale_order_type/__manifest__.py]: This should be version `13.0.1.0.0`

**Review by joao-p-marques — COMMENTED** [review]
@oihane code review and looks good :+1: Just a minor comment...

File: website_sale_order_type/__manifest__.py (hunk 5 above)
Comment (oihane) [reply website_sale_order_type/__manifest__.py]: Done, thanks

**Review by joao-p-marques — APPROVED** [review]
LGTM code-wise :+1:

**Review by chienandalu — APPROVED** [review]

**Pull request merged**